src/actix/api/search_api.rs

```diff
@@ -27,6 +28,11 @@ async fn search_points(
     params: Query<ReadParams>,
     ActixAccess(access): ActixAccess,
 ) -> HttpResponse {
+    let pass = match request.check(&dispatcher, &access, &collection.name).await {
+        Ok(pass) => pass,
+        Err(err) => return process_response_error(err, Instant::now()),
```

Comment:
In order to prevent the usage of `async {}`, we return a 'dummy' time value here. 
If the timing information is important for errors too, I'll move it into an async block.

Reply:
Can't we just instantiate the time right at the start of the function, and pass it in here, without async?

Reply:
You mean simply putting the `Instant::now()` call before strict-mode checking to measure it?
What about `helpers::time`, should we replace this with a custom implementation and pass the `Instant` from the start of the function too?